thinkhazard_processing/models.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+from sqlalchemy import (Column, ForeignKey,
+                        Boolean, Date, Integer, String)
+from sqlalchemy.ext.declarative import declarative_base
+import geoalchemy2
+
+
+Base = declarative_base()
+
+
+class Administrativedivision(Base):
+    __tablename__ = 'administrativedivision'
+    __table_args__ = {u'schema': 'processing'}
+    id = Column(Integer, primary_key=True)
+    code = Column(Integer, nullable=False, unique=True)
+    leveltype_id = Column(Integer, nullable=False)
+    name = Column(String, nullable=False)
+    parent_code = Column(Integer,
+                         ForeignKey('processing.administrativedivision.code'))
+    geom = Column(geoalchemy2.Geometry('MULTIPOLYGON',
+                                       3857,
+                                       management=True))  # bbox ?
+
+
+class Dataset(Base):
+    __tablename__ = 'dataset'
+    __table_args__ = {u'schema': 'processing'}
+
+    # hazard_set is the ID common to the 3 layers,
+    # as reported by geonode (hazard_set field)
+    hazard_set_id = Column(String(1024), primary_key=True)  # eg: "EQ-PA"
+    # rating 1 is the rating coming from the metadata
+    rating1 = Column(Integer)
```

Comment:
There are no such fields for now, eg in http://45.55.174.20/api/layers/88/

I propose
```
    calculation_method_quality = Column(Integer) # 0 to 10
    scientific_quality = Column(Integer) # 0 to 2
```